### ISSUE TITLE
Support for `stubArray`, `stubFalse`, `stubObject`, `stubString`, `stubTrue`

### ISSUE THREAD

**Issue by chhw130** [toss/es-toolkit#626]
Hello, @raon0211 
Can I work `stubArray`, `stubFalse`, `stubObject`, `stubString`, `stubTrue` of #91 ??

**Comment by raon0211** [comment]
Yes! Can you add them in the compatibility library? Probably some of them would be in the `src/compat/_internal` directory...

**Comment by raon0211** [comment]
The functions were implemented here:

- https://es-toolkit.dev/reference/compat/util/stubArray.html
- https://es-toolkit.dev/reference/compat/util/stubFalse.html
- https://es-toolkit.dev/reference/compat/util/stubObject.html
- https://es-toolkit.dev/reference/compat/util/stubString.html
- https://es-toolkit.dev/reference/compat/util/stubTrue.html
